dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.6_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -57,3 +57,8 @@ d//
 delimiter ;
 CALL uc_dolphin_T_t_ds_alert_R_sign;
 DROP PROCEDURE uc_dolphin_T_t_ds_alert_R_sign;
+
+-- add unique key to t_ds_relation_project_user
+ALTER TABLE t_ds_relation_project_user ADD UNIQUE KEY uniq_uid_pid(user_id,project_id);
```

Comment:
i think we should add to new SQL directory named `3.0.0`?

Reply:
> i think we should add to new SQL directory named `3.0.0`?

It's a bug fix. I think it should be included in 2.0.6-release. WDYT? @zhongjiajie

Reply:
In general, I think change targe branch with `dev` database change should add to 3.0.0_schema, and committer judge whether should release in bug fix version. If it is only put in 2.0.5 and not in 3.0.0, it may be that the 3.0.0-bate release will miss this commit.

But in case, `dev` branch without `3.0.0_schema` folder,(and that is my wrong, i forgot to add it after 3.0.0-release) so i think it could change into 2.0.6 or 3.0.0

Reply:
> In general, I think change targe branch with `dev` database change should add to 3.0.0_schema, and committer judge whether should release in bug fix version. If it is only put in 2.0.5 and not in 3.0.0, it may be that the 3.0.0-bate release will miss this commit.
> 
> But in case, `dev` branch without `3.0.0_schema` folder,(and that is my wrong, i forgot to add it after 3.0.0-release) so i think it could change into 2.0.6 or 3.0.0

I agree with you. @zhongjiajie